File: Leetcode/3223. Minimum Length of String After Operations.py
from collections import Counter
class Solution:
    def minimumLength(self, s: str) -> int:
        counter=Counter(s)
        res=0
        for j in counter:
            #print(i)
            if (counter[j]&1)==1:
                res+=1
            else:
                res+=2
        return res

        

File: Leetcode/3223. Minimum Length of String After Operations.cpp
class Solution {
public:
    int minimumLength(string s) {
        unordered_map<char,int> m;
        int res=0;
        for(char c:s) m[c]++;
        for(auto i=m.begin();i!=m.end();i++){
            if(i->second&1){
                res+=1;
            }else{
                res+=2;
            }
        }
        return res;
    }
};
